translations/es.ts

```diff
@@ -2028,7 +2028,7 @@ Compártelo con tus amigos para poder comunicarte.</translation>
         <location filename="../src/persistence/toxsave.cpp" line="58"/>
         <source>Warning: you&apos;ve chosen a file that is not a Tox save file; ignoring.</source>
         <comment>popup text</comment>
-        <translation>Advertencia: el archivo seleccionado no es un archivo Tox. Ignorado.</translation>
+        <translation>Advertencia: El archivo seleccionado no es un archivo Tox. Ignorando.</translation>
```

Comment:
Even if el may not be wrong, neither is El. (same reference). And It looks better.
Wrt to ignorando, I chose it because it's closer to the original.
Of course, both work. So if you prefer ignored over ignoring, I can undo that part.